public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+
+             1111111101             
+        MSB <----------> LSB                                
+
+        +    0000001000  
+        ====================
+             0000000101
+        MSB <----------> LSB 
+
+        +     OVERFLOW
+
+The overflow would toggle the `BCU` on, causing the heretofore ineffective `TBC` to transfer control to the address in the (A) field, which was `0`. If Ed Nather was not familiar with the overflow aspect of the `BCU`, then his reading of the code would indeed lead to the diagnosis of a loop without a test. A standard conditional jump consisted of some test, followed by a `TBC` instruction, which would `JUMP` out of the loop only if the test had succeeded. It was quite natural, then, for a RPC-4000 programmer to come across a "free floating" `TBC` instruction, with no preceding test and conclude:
+
+        But the loop had no test in it.
+
+This scenario seems closer to the original story: The `JUMP` is there, as well as the overflow and the seemingly unnecessary `1` in the index register bit. There are only two deviations from the original account:
+
+1. The opcode is never modified.
+2. The magical nature of the hack is due not only to Mel's prowess, but also - perhaps mostly - to Ed Nather's incomplete understanding of the machine. Had he known that the `TBC` instruction was influenced by an overflow, he would have cracked the problem right away, leaving no story for posterity. 
+
+Anyone who likes coding can imagine Mel Kaye's reaction, when he read about the path of an overflow triggering a jump. The challenge was immediately obvious. Having run a trivial POC, he naturally proceeded to the next level, causing the overflow at the end of an iterative process, after the data had been drained. This way, he could avoid the standard test at the end of the loop. The stroke of genius was completed when he managed to usefully incorporate this structure into a real-world program. 
+
+### So What You're Saying is...
+
+
+Whichever version you prefer - the number juggling trick or the deviously elegant `TBC` manipulation - it is clear that Ed Nather's account was rooted in faulty memory (and probably a faulty understanding of RPC-4000). This finding does not diminish the story's charm. Most developers can identify with the laborious excavation through another programmer's "impossible" code. The charm of a self-modifying program is still there and the hack, in both implementations, is impressive. 
+
+If this analysis can cast a shadow on the myth of Mel, it is not related to Nather's credibility, nor to Mel's undeniable skill. Rather, it stems from the obvious lack of real value that the hack added to the program. It's very unlikely that a standard loop would have degraded the program's performance in any noticeable way. Mel's testless loop was clearly a vain addition of complexity, in an environment that was very complex on its own.
```

Comment:
This paragraph contains a certain light which I would appreciate reading into the exposition to the article. It foreshadows the path of the article, and gives a certain non-tech understanding for the tech discussion, which the reader delves into here.